sandbox.go

```diff
@@ -899,9 +900,10 @@ func (sb *sandbox) clearNetworkResources(origEp *endpoint) error {
 }
 
 func (sb *sandbox) isEndpointPopulated(ep *endpoint) bool {
-	sb.Lock()
+	sb.RLock()
+	defer sb.RUnlock()
+
 	_, ok := sb.populatedEndpoints[ep.ID()]
```

Comment:
The access to this map is already protected by sandbox lock.
This change does not help.

The root cause of the issue was a delete on the map run outside of sandbox lock and was fixed in #1512